internal/cli/cmd/util/error.go

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
this file should not be named with error.go, the functions in this file is doing some validation, maybe name it with validator.go is a better choice